### ISSUE TITLE
Autocomplete inputs

### ISSUE THREAD

**Issue by bvorak** [Herder-IT-Solutions/HerderBib#20]
Autocomplete bei:
- Name
- Benutzername
- ID
- etc...

**Comment by ghost** [comment]
We could do it when the database group make a SQL enquiry that give an Array of all isbns, names etc... back that begin with the transfered beginning. 
The SQL enquiry should be easy with the Like-Operator and the % after it.

**Comment by cnmicha** [comment]
getStudentsByFirstNamePattern and getStudentsByLastNamePattern already exist in TDBConnection :)

**Comment by ghost** [comment]
@cnmicha Yes but for autocomplete i guess that you dont have the full first name or last name or isbn ... and you want that if only one possible object exist everything will fill out automaticaly.
When you have to type in the whole isbn to get the information it is not so effetive in my opinion.
With the like Operator you can get an array of objects that begin with the parameter and of the length of the array is one we can autofill everthing. Should we make this function ?

**Comment by cnmicha** [comment]
??? @ghgd-mp

**Comment by cnmicha** [comment]
getStudentByLDAPUserPattern was implemented in 77dad8bf08dcec34aa730d616c5df7404d54e22f

**Comment by ghost** [comment]
getStudentByLDAPUserPattern was implemented 
for autocomplete use %-Operator
